### PR TITLE
Add ra-enterprise packages to Ecosystem documentation

### PR DESCRIPTION
**Pull request by djhi**
image

### CODE REVIEW

**Review by Luwangel — COMMENTED** [review]
I let you some suggestions. It's not only typos so you don't have to apply everything.

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -45,9 +50,10 @@ See the [Data Provider](./DataProviders.md#available-providers) page.
 
 ## Miscellaneous
 
-- [marmelab/ra-tree-ui-materialui](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/): Components to show data represented as a tree. Only compatible with react-admin v2 for now.
-- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/): Components providing the logic but no UI to show data represented as a tree. Only compatible with react-admin v2 for now.
-- [ra-customizable-datagrid](https://github.com/fizix-io/ra-customizable-datagrid): plugin that allows to hide / show columns dynamically.
+- [@react-admin/ra-preferences](https://marmelab.com/ra-enterprise/modules/ra-preferences) <img class="icon" src="./img/premium.svg" /> An [ra-enterprise](https://marmelab.com/ra-enterprise) module which persists user preferences (language, theme, filters, datagrid columns, sidebar position, etc) in local storage.
```
[the remark this answers is not in the document]
Comment (djhi) [reply docs/Ecosystem.md:53]: I wanted a link to the ra-enterprise home link as well

File: docs/List.md [hunk 2]
```diff
@@ -2395,6 +2395,16 @@ export const CommentList = (props) => (
 
 As you can see, nothing prevents you from using `<Field>` components inside your own components... provided you inject the current `record`. Also, notice that components building links require the `basePath` component, which is also available from `useListContext`.
 
+## Third-Party Components
+
+You can find components for react-admin in third-party repositories.
+
+- <img class="icon" src="./img/premium.svg" /> [ra-editable-datagrid](https://marmelab.com/ra-enterprise/modules/ra-editable-datagrid). A [ra-enterprise](https://marmelab.com/ra-enterprise) component which offers an "edit-in-place" experience in a <Datagrid>.
+- <img class="icon" src="./img/premium.svg" /> [ra-tree](https://marmelab.com/ra-enterprise/modules/ra-tree). Tree hooks and components from the [ra-enterprise](https://marmelab.com/ra-enterprise) package. This module is agnostic as to how you store the tree structure in the backend side.
```
[the remark this answers is not in the document]
Comment (djhi) [reply docs/List.md]: I'll keep what @fzaninotto wrote in the `ra-tree` package

**Review by Luwangel — APPROVED** [review]

File: docs/Ecosystem.md [hunk 3]
```diff
@@ -5,13 +5,20 @@ title: "Ecosystem"
...
 # Ecosystem
 
-- [Inputs and Fields](#inputs-and-fields)
-- [Translations](#translations)
-- [Authentication Providers](#authentication-providers)
-- [Authorization Management](#authorization-management)
-- [Data Providers](#data-providers)
-- [User Interface](#user-interface)
-- [Miscellaneous](#miscellaneous)
+- [Ecosystem](#ecosystem)
```
Comment (fzaninotto) [inline docs/Ecosystem.md:8]: Why did you add a link to the page itself?

File: docs/List.md [hunk 4]
```diff
@@ -2395,6 +2395,16 @@ export const CommentList = (props) => (
 
 As you can see, nothing prevents you from using `<Field>` components inside your own components... provided you inject the current `record`. Also, notice that components building links require the `basePath` component, which is also available from `useListContext`.
 
+## Third-Party Components
+
+You can find components for react-admin in third-party repositories.
+
+- [ra-editable-datagrid](https://marmelab.com/ra-enterprise/modules/ra-editable-datagrid) <img class="icon" src="./img/premium.svg" />: a [ra-enterprise](https://marmelab.com/ra-enterprise) component which offers an "edit-in-place" experience in a `<Datagrid>`.
+- [ra-tree](https://marmelab.com/ra-enterprise/modules/ra-tree) <img class="icon" src="./img/premium.svg" />: tree hooks and components from the [ra-enterprise](https://marmelab.com/ra-enterprise) package. This module is agnostic as to how you store the tree structure in the backend side.
+- [marmelab/ra-tree-ui-materialui](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/): components to show data represented as a tree. Only compatible with react-admin v2.
```
Comment (fzaninotto) [inline docs/List.md:2404]: no need to mention it as it's v2 only

File: docs/List.md [hunk 5]
```diff
@@ -2395,6 +2395,16 @@ export const CommentList = (props) => (
...
+- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/): components providing the logic but no UI to show data represented as a tree. Only compatible with react-admin v2.
```
Comment (fzaninotto) [inline docs/List.md:2405]: same

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Ecosystem.md (hunk 3 above)
Comment (djhi) [reply docs/Ecosystem.md:8]: I don't even remember adding it ^^

**Review by fzaninotto — APPROVED** [review]
